.ci.yaml

```diff
@@ -5481,15 +5481,20 @@ targets:
         ["devicelab", "hostonly", "mac"]
       task_name: hot_mode_dev_cycle_ios_simulator
 
-  - name: Mac_ios hot_mode_dev_cycle_ios_xcode_debug
```

Comment:
I'm supportive of the rename but note that this will need to get cherry picked if you want these tests to run on beta/stable branches (I'm guessing important here?)